app/models/plugin.ts

```diff
@@ -20,7 +12,7 @@ export type ProgramOutput = string | boolean;
 /**
  * Represents a function which converts graphs into programs or else returns an error that has toString.
  */
-export type Interpreter = (graph: Graph) => Promise<Program>;
+export type Interpreter = (graph: any) => Promise<Program>;
```

Comment:
It will be changed from any once serialization is reworked after the plugin typing update. This type is supposed to be a serialized graph.